### PR TITLE
Update tuya.js

### PR DESCRIPTION
**Pull request by kfir-bar**
My first :) change for the item - https://neatho.me/en/catalog/smart-gadget/controller/tuya-zigbee-smart-knob-wireless-scene-switch-button/
* I didn't attach a picture.

### CODE REVIEW

File: devices/tuya.js [hunk 1]
```diff
@@ -1782,4 +1782,14 @@ module.exports = [
...
     },
+    {
+        fingerprint: [{modelID: 'TS004F', manufacturerName: '_TZ3000_4fjiwweb'}],
+        model: 'ERS-10TZBVK-AA',
+        vendor: 'TuYa',
+        description: 'Smart knob',
+        fromZigbee: [fz.command_step, fz.command_toggle, fz.command_move_hue],
+        toZigbee: [],
+        exposes: [e.action(['button_*_hold', 'button_*_single', 'button_*_double', 'button_*_triple', 'button_*_quadruple',
+            'button_*_release'])],
```
Comment (Koenkk) [inline devices/tuya.js:1793]: Please replace the `*` with the actual values the are being published as `action`

**Comment by kfir-bar** [comment]
Where can I find the exact definition? I am not familiar with it.

**Comment by Koenkk** [comment]
@kfir-bar you can check what is published when you press buttons on the device (e.g. `{"action": "button_1_hold"}`)

**Comment by Koenkk** [comment]
Thanks!

**Comment by kfir-bar** [comment]
Thank you!

**Comment by ekesgin** [comment]
Hey guys, I got this device from China today.  When will it be merged into master?

**Comment by Koenkk** [comment]
1 january

**Pull request merged**